pkg/scalers/gcp_stackdriver_scaler.go

```diff
@@ -109,6 +110,18 @@ func parseStackdriverMetadata(config *ScalerConfig, logger logr.Logger) (*stackd
 		meta.activationTargetValue = activationTargetValue
 	}
 
+	if val, ok := config.TriggerMetadata["valueIfNull"]; ok {
+		if val == "" {
+			meta.valueIfNull = nil
+		} else {
+			valueIfNull, err := strconv.ParseFloat(val, 64)
+			if err != nil {
+				return nil, fmt.Errorf("valueIfNull parsing error %w", err)
+			}
+			meta.valueIfNull = &valueIfNull
+		}
+	}
```

Comment:
```suggestion
	if val, ok := config.TriggerMetadata["valueIfNull"]; ok && val != "" {
		valueIfNull, err := strconv.ParseFloat(val, 64)
		if err != nil {
			return nil, fmt.Errorf("valueIfNull parsing error %w", err)
		}
		meta.valueIfNull = &valueIfNull
	}
```

Reply:
done

Reply:
did you push it? xD

Reply:
I meant this suggestion to reduce the complexity